src/ports/postgres/modules/deep_learning/madlib_keras_validator.py_in

```diff
@@ -45,6 +46,8 @@ from utilities.validate_args import input_tbl_valid
 from utilities.validate_args import output_tbl_valid
 from madlib_keras_wrapper import parse_and_validate_fit_params
 from madlib_keras_wrapper import parse_and_validate_compile_params
+from madlib_keras_wrapper import losses
```

Comment:
We should use `import keras.losses as losses` just to keep it consistent with the next line.